sympy/functions/special/tests/test_delta_functions.py

```diff
@@ -12,13 +12,18 @@
 from sympy.utilities.misc import filldedent
 
 x, y = symbols('x y')
-
+i = symbols('t',nonzero=True)
+j = symbols('j',positive=True)
+k = symbols('k',negative=True)
```

Comment:
Please add a space after the comma, as per pep8 recommendations and then this is good to go in. +1